### DISCUSSION TITLE
How to compile the developer version?

### DISCUSSION THREAD

**Discussion by Tyler887** [ThioJoe/YT-Spammer-Purge#814] [Q&A]
I recently read #793 which says there are no actual releases for it. However, I cannot run the proper command to install a dependency:
```
pip3 install googleapiclient
```
This returns:
```
ERROR: Could not find a version that satisfies the requirement googleapiclient (from versions: none)
ERROR: No matching distribution found for googleapiclient
```
and that's because the page on the [Python Package Index](https://pypi.org) - https://pypi.org/project/googleapiclient/ - leads me to a 404 page.
So how do I even compile the dev version and install `googleapiclient`?

Or we could do something like this and put it in the code:
```python
class HttpError(Exception):
   pass
```

**Comment by Kurdakov** [comment]
there is requirements.txt in dev version to install all needed dependencies

so

pip install -r requirements.txt

if you open requirements.txt you'd see that package name is actually
google_api_python_client and not googleapiclient

**Comment by Kurdakov** [comment]
concerning your question in another thread - to make an exe you need to use either pyinstaller or nuitka, 

I think that it's something like:
pip install pyinstaller
pyinstaller --onefile  YTSpammerPurge.py 

for nuitka (it will produce bunch of dlls for windows as static python libs on windows are not supported, also probably you'd need to move some not found files from installed packages to YTSpammerPurge directory manually)  take a look here https://github.com/ThioJoe/YT-Spammer-Purge/discussions/258

otherwise why not just run 
python YTSpammerPurge.py
